src/OgRoleInterface.php

```diff
@@ -51,4 +51,42 @@
    */
   public function setId($id);
 
+  /**
+   * Returns default properties for the given default OG role name.
+   *
+   * These are the properties used to create the two roles that are required by
+   * every group: the 'member' and 'non-member' roles.
```

Comment:
Apparently I also found this confusing yesterday, in #238 this method is removed and replaced by a protected method `GroupManager::getRequiredDefaultRoles()` :)

See https://github.com/amitaibu/og/pull/238/files#diff-241b87c62faf75c9ea711421b2899ea9R366

I'll backport that change to this issue.